src/main/java/com/eppo/sdk/helpers/ConfigurationStore.java

```diff
@@ -13,34 +13,38 @@
 /**
  * Configuration Store Class
  */
+@Slf4j
```

Comment:
This annotation magically (😞 ) makes the `log` method available.

(Side note: I'm slightly more cool with this particular use case and side effect)